spec/javascripts/modules/track-responses.spec.js

```diff
@@ -82,6 +82,17 @@ describe('Track responses', function () {
       )
     })
 
+    it('filters out sensitive responses', function () {
```

Comment:
The tests you've added appear to be the same but in different contexts, is that correct? Just wanted to check are they all necessary?

Reply:
Yes you're right, two are the same - the third one (select) is subtlety different. I wanted to test this only once - ideally - however the tests are structured around input type (radio, checkbox and select). Both of these caused me to duplicate it. Would testing this just once - in a separate `describe` block - be ok and work for all input types? That would be my preference.

Reply:
Thanks for the explanation. No, it sounds sensible the way you've done it, thanks.